requirements.txt

```diff
@@ -1,7 +1,19 @@
-Django==1.8.4
-django-extensions
+Django==1.8.2
```

Comment:
We should be going forward in versions, not backward. We probably want to be on 1.8.5, this is the current bug-fix release of 1.8.